pkgpanda/__init__.py

```diff
@@ -903,7 +903,7 @@ def _get_service_names(_dir):
                     make_directory(state_dir_path)
                     if package.username and not is_windows:
                         uid = sysusers.get_uid(package.username)
-                        _check_call(['chown', '-R', str(uid), state_dir_path])
+                        check_call(['chown', '-R', str(uid), state_dir_path])
```

Comment:
This reverts logging for `call` introduced in https://github.com/dcos/dcos/pull/6910